extensions/sql-database-projects/src/models/project.ts

```diff
@@ -823,6 +823,55 @@ export class Project implements ISqlProject {
 		await this.addToProjFile(sqlCmdVariableEntry);
 	}
 
+	/**
+	 * Adds DatabaseSource property to the project
+	 * @param databaseSource Source of the database to add
+	 */
+	public async addDatabaseSourceToProjFile(databaseSource: string): Promise<void> {
```

Comment:
I'll let maintainers to comment. I'm not sure about method names. Seems like majority of `..ToProjFile` members are private and what's exposed publicly is more object-oriented, for example `changeTargetPlatform`. Maybe this )and below) should be called just `appendDatabaseSource`?

Reply:
I'm ok either way, no strong opinions on this.

Reply:
I've renamed this to `addDatabaseSource` and `removeDatabaseSource` as originally suggested in the issue.  I think this is clearer than using 'append'.